### ISSUE TITLE
Review XML Form Builder forms and remove irrelevant forms

### ISSUE THREAD

**Issue by jefferya** [cwrc/Islandora-CWRC-Writer#35]
Some XML Form Builder forms are automatically added by this module - should they be removed @ilovan @SusanBrown ?

https://github.com/cwrc/Islandora-CWRC-Writer/tree/master/data/forms

Also see the XML Form Builder Form associations and remove associations as needed: 

/admin/islandora/xmlform/associations

I don't believe these ever worked properly, were never maintained, and are obsolete with the move to TEI, correct?

**Comment by ilovan** [comment]
If these are the old, ARC produced entity creation forms that were supposed to dinamically change with the schema, then yes, by all means, they should be removed

**Comment by SusanBrown** [comment]
confirmed

On Jun 5, 2018, at 6:12 PM, Mihaela Ilovan <notifications@github.com<mailto:notifications@github.com>> wrote:


If these are the old, ARC produced entity creation forms that were supposed to dinamically change with the schema, then yes, by all means, they should be removed

**Comment by jefferya** [comment]
These were produced by DGi (assuming for EMiC but enabled in CWRC as well)

**Comment by SusanBrown** [comment]
Is this done @jefferya ? Mihaela thinks it might be.

**Comment by jefferya** [comment]
@SusanBrown @ilovan No. Can this be added to Nia's list for the programming portion? Mihaela and/or others can do the second part from the Drupal interface.
